src/pages/wheelcalc.tsx

```diff
@@ -0,0 +1,165 @@
+  }))
+
+  // Calculations.
+  const wheelCost = data.wheelCredit ? 5 : 10;
+
+  let coinsTotal = 0;
+  let spinsPerDayString = "";
+  let coinsPerDayString = "";
+		
+  if (data.numSpins % data.numDays !== 0) {
+    let spinsPerDay = Math.floor(data.numSpins / data.numDays);
+    let extraSpinDays = data.numSpins % data.numDays;
+    let regularDays = data.numDays - extraSpinDays;
+    spinsPerDayString =  (spinsPerDay + 1).toLocaleString() + " spins per day for " + extraSpinDays.toLocaleString() + " days, and " +  spinsPerDay.toLocaleString() + " for " + regularDays.toLocaleString() + " days";
+
+    let coinsPerDay = 0;
+    let coinsPerExtraSpinDay = 0;
+    for (let i = 0; i < (spinsPerDay + 1); i++)
+    {
+        coinsPerExtraSpinDay += i*wheelCost;
+    }
+    for (let i = 0; i < spinsPerDay; i++)
+    {
+        coinsPerDay += i * wheelCost;
+    }
+
+    coinsPerDayString = coinsPerExtraSpinDay.toLocaleString() + " coins per day for " + extraSpinDays.toLocaleString() + " days, and " + coinsPerDay.toLocaleString() + " for " + regularDays.toLocaleString() + " days";
+    coinsTotal = coinsPerDay * regularDays + coinsPerExtraSpinDay * extraSpinDays;
+  } else {
+    let spinsPerDay = data.numSpins / data.numDays;
+    spinsPerDayString =  spinsPerDay.toLocaleString() + " spins per day";
+
+    let coinsPerDay = 0;
+    for (let i = 0; i < spinsPerDay; i++)
+    {
+        coinsPerDay += i * wheelCost;
+    }
+    coinsPerDayString = coinsPerDay.toLocaleString() + " coins per day";
+    coinsTotal = coinsPerDay * data.numDays;
+  }
+
+  const chestsWithoutExploring = Math.ceil(coinsTotal / 100);
+  
+  let coinsPerOJ = 0.0;
+  let coinsPerCider = 0.0;
+  let coinsPerLemonade = 0.0;
+  let coinsPerAP = 0.0;
+    
+  let chestsPerOJ = 0.0;
+  let chestsPerCider = 0.0;
+  let chestsPerLemonade = 0.0;
+  let chestsPerAP = 0.0;
+  
+  if (data.runecube) {	
+    coinsPerOJ = 1/1.1;
+    coinsPerCider = 9.11;
+    coinsPerLemonade = 1/2.74;
+    coinsPerAP = 9.11;
+
+    chestsPerOJ = 1/65.91;
+    chestsPerCider = 1/6.56;
+    chestsPerLemonade = 1/163.96;
+    chestsPerAP = 1/6.56;
+  } else {	
+    coinsPerOJ = 1/1.27;
+    coinsPerCider = 7.89;
+    coinsPerLemonade = 1/4.56;
+    coinsPerAP = 7.89;
+
+    chestsPerOJ = 1/84;
+    chestsPerCider = 1/8.36;
+    chestsPerLemonade = 1/208.95;
+    chestsPerAP = 1/8.36;
+  }
+  
+  const ojNeeded = Math.ceil(coinsTotal/(coinsPerOJ + (chestsPerOJ * 100)));
+  const ciderNeeded = Math.ceil(coinsTotal/(coinsPerCider + (chestsPerCider * 100)));
+  const lemonadeNeeded = Math.ceil(coinsTotal/(coinsPerLemonade + (chestsPerLemonade * 100)));
+  const apNeeded = Math.ceil(coinsTotal/(coinsPerAP + (chestsPerAP * 100)));
+  
+  const ojNeededString = ojNeeded.toLocaleString() + " OJ (" + Math.round(ojNeeded * coinsPerOJ).toLocaleString() + " coins through exploring & " + Math.round(ojNeeded * chestsPerOJ).toLocaleString() + " chests)";
```

Comment:
Up to you if you want to stick to one string but the text input widget supports a `tooltip` string as well and I use that in some places to show extra data while keeping the main output more direct.